alot/commands/globals.py

```diff
@@ -733,7 +733,8 @@ def apply(self, ui):
                           priority='error')
                 return
             try:
-                self.envelope.parse_template(open(path).read())
+                with open(path) as f:
+                    self.envelope.parse_template(f.read())
```

Comment:
Is it the pythonic way to use a with block even if the opened object is only used once? If I remember correctly all open objects are automatically closed on destruction and if there is no reference to them any longer (like in all these one line usages) there are no further references/names. Or are these objects only cleaned up when the next block or even the only if the function is finished?